caf-audit-service-container/pom.xml

```diff
@@ -132,6 +132,10 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+        </dependency>
```

Comment:
- `jakarta.annotation-api`
- This is the "Testing dependencies" section.  Move to the correct location.